src/Components/CreateImageWizardV2/steps/Repositories/Repositories.tsx

```diff
@@ -320,7 +320,11 @@ const Repositories = () => {
 
   const handleSelectAll = () => {
     if (data) {
-      updateSelected(data.data?.map((repo) => repo.url) || []);
+      updateSelected(
+        data.data
+          ?.filter(({ status }) => status === 'Valid')
+          .map((repo) => repo.url) || []
+      );
```

Comment:
Indeed, and now that you mention it, it is an incomplete bug fix. I need to make sure the that selectAll button is greyed out if none of the repositories are valid as well...

Reply:
I think I could easily rewrite the repositories piece to match our own new template wizard which is very similar. 
If you'd like me to maintain it I'd suggest doing that so the look/feel is identical between both wizards for that step.
Also because of the similarity in syntax between this flavour of Redux and react-query I think it would be an easy lift.

Reply:
Feel free to throw-up a card and ping me on it if you'd like me to do that.